django_downloadview/middlewares.py

```diff
@@ -75,14 +75,10 @@ def is_download_response(self, response):
         whose file attribute have either an URL or a file name.
 
         """
-        if super(RealDownloadMiddleware, self).is_download_response(response):
-            try:
-                return response.file.url or response.file.name
-            except AttributeError:
-                return False
-            else:
-                return True
-        return False
+        return (
+            super().is_download_response(response)
+            and (hasattr(response.file, 'url') or hasattr(response.file, 'name'))
```

Comment:
The original version of this also wants the properties to be truthy, so I think `getattr(response.file, ..., None)` would be better.

Reply:
That may be good, I was going in the pre-condition that: if the file _has_ one of those attribute that attribute was going to be a truthy value.

I have updated the branch with also a `bool` casting to be coherent with the type returned by base class.

Reply:
Makes sense, thanks.